managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/PositionImpl.java

```diff
@@ -129,6 +129,20 @@ public boolean equals(Object obj) {
         return false;
     }
 
+    public static PositionImpl convertStringToPosition(String positionString) {
+        if (positionString == null) {
+            throw new NullPointerException();
+        } else {
+            String[] strings = positionString.split(":");
+            if (strings.length != 2) {
+                throw new IndexOutOfBoundsException();
+            }
+            long ledgerId = Long.parseLong(strings[0]);
+            long entryId = Long.parseLong(strings[1]);
```

Comment:
can you please catch "NumberFormatException" and then rethrow a meaningful error message ?